ChallengeTestRunner/challenge_test_lib/challenge_test.py

```diff
@@ -380,6 +381,34 @@ def challenge_fun(self, *args, **kwargs):
             raise err
         return self.__class__.CHALLENGE_FUN(*args, **kwargs)
 
+    @functools.lru_cache()
+    def str_dist(self, s, t):
+        # http://rosettacode.org/wiki/Levenshtein_distance#Memoized_recursion
+        if not s:
+            return len(t)
+        if not t:
+            return len(s)
+        if s[0] == t[0]:
+            return self.str_dist(s[1:], t[1:])
+        l1 = self.str_dist(s, t[1:])
+        l2 = self.str_dist(s[1:], t)
+        l3 = self.str_dist(s[1:], t[1:])
+        return 1 + min(l1, l2, l3)
+
+    def assert_similar(self, string1, string2, distMax=1, caseSensitive=False):
+        if not string1:
+            return len(string2)
+        if not string2:
+            return len(string1)
+        if caseSensitive == True:
+            string1 = string1.lower()
+            string2 = string2.lower()
+        dist = self.str_dist(string1,string2)
+        if(dist > distMax):
+            msg = self._formatMessage("Strings não são similares.")
```

Comment:
Utilizar a mensagem padrão apenas se não for recebida outra mensagem pelo método (deveria receber um argumento opcional `msg` (veja os outros asserts).